app/grandchallenge/archives/serializers.py

```diff
@@ -13,6 +31,7 @@ class ArchiveSerializer(serializers.ModelSerializer):
     url = URLField(source="get_absolute_url", read_only=True)
     # Include the read only name for legacy clients
     name = ReadOnlyField()
+    items = ArchiveItemSerializer(many=True, read_only=True)
```

Comment:
It's better not to add items at all to this serializer, if the user needs this then they should be able to get the information from the archive item list endpoint.

Reply:
You will need to add an archive item list endpoint with permissions, filters and tests to do this!